content/en/apps/reference/app-settings/transitions.md

```diff
@@ -575,3 +575,48 @@ Supported `events_types` are:
   }
 ]
 ```
+
+### update_clinics
+Adds a contact’s info to a data record so as to attribute an incoming SMS message or report to the appropriate contact. 
+
+#### Configuration
+As of version 3.12 you can add configuration to send a message whenever a contact match fails while running this transition. Configuration is stored in the `update_clinics` field of app_settings.json as a list of objects connecting forms to messages. Every object should have this structure:
+
+| Property | Description |
+|---|---|
+| `form` | Form code. **Required** |
```

Comment:
What does "required" mean here? Does the transition fail to load if this field is not set?

Reply:
Removed it to avoid confusion.